.pre-commit-config.yaml

```diff
@@ -5,76 +5,90 @@ repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.3.0
     hooks:
-      - id: trailing-whitespace
+      - id: fix-byte-order-marker
+      - id: mixed-line-ending
       - id: end-of-file-fixer
-        exclude: ^.*\.egg-info/
+        exclude: \.egg-info/[^/]+
+      - id: trailing-whitespace
       - id: check-merge-conflict
       - id: check-case-conflict
+      - id: check-symlinks
+      - id: destroyed-symlinks
       - id: check-json
       - id: check-toml
-        exclude: tests/fixtures/invalid_lock/poetry\.lock
+        exclude: fixtures/invalid_lock
       - id: check-yaml
       - id: pretty-format-json
         args: [--autofix, --no-ensure-ascii, --no-sort-keys]
+      - id: check-vcs-permalinks
       - id: check-ast
       - id: debug-statements
       - id: check-docstring-first
+      - id: name-tests-test
+        args: [--pytest-test-first]
+        exclude: (compat|helpers|types|utils)\.py|fixtures/[^/]+
+      - id: no-commit-to-branch
 
   - repo: https://github.com/pre-commit/pygrep-hooks
     rev: v1.9.0
     hooks:
+      - id: python-check-blanket-noqa
+      - id: python-check-blanket-type-ignore
       - id: python-check-mock-methods
+      - id: python-no-eval
+      - id: python-no-log-warn
+        exclude: fixtures/[^/]+
       - id: python-use-type-annotations
-      - id: python-check-blanket-noqa
+
+  - repo: https://github.com/pre-commit/pre-commit
+    rev: v2.20.0
+    hooks:
+      - id: validate_manifest
 
   - repo: https://github.com/asottile/yesqa
     rev: v1.4.0
     hooks:
       - id: yesqa
         additional_dependencies: &flake8_deps
-          - flake8-annotations==2.9.0
-          - flake8-broken-line==0.5.0
-          - flake8-bugbear==22.7.1
+          - flake8-annotations==2.9.1
+          - flake8-broken-line==0.6.0
```

Comment:
I'm not 100% sure, but I think that if the code is formatted by `black`, those things that `flake8-broken-line` checks will never appear.

Reply:
It looks like you are correct: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html, ctrl-f `backslashes`.